.gitignore

```
@@ -3,3 +3,4 @@
 
 venv/
 .idea
+.values.yaml.swp
```

This can be any `.swp` file as it comes from vim:
```suggestion
*.swp

```